examples/observability/eks-cluster-with-adot-amp-grafana-for-haproxy/main.tf

```
@@ -0,0 +1,146 @@
+provider "aws" {
+  region = local.region
+}
+
+provider "kubernetes" {
+  host                   = module.eks_blueprints.eks_cluster_endpoint
+  cluster_ca_certificate = base64decode(module.eks_blueprints.eks_cluster_certificate_authority_data)
+
+  exec {
+    api_version = "client.authentication.k8s.io/v1alpha1"
+    command     = "aws"
+    # This requires the awscli to be installed locally where Terraform is executed
+    args = ["eks", "get-token", "--cluster-name", module.eks_blueprints.eks_cluster_id]
+  }
+}
+
+provider "helm" {
+  kubernetes {
+    host                   = module.eks_blueprints.eks_cluster_endpoint
+    cluster_ca_certificate = base64decode(module.eks_blueprints.eks_cluster_certificate_authority_data)
+
+    exec {
+      api_version = "client.authentication.k8s.io/v1alpha1"
+      command     = "aws"
+      # This requires the awscli to be installed locally where Terraform is executed
+      args = ["eks", "get-token", "--cluster-name", module.eks_blueprints.eks_cluster_id]
+    }
+  }
+}
+
+provider "grafana" {
+  url  = var.grafana_endpoint
+  auth = var.grafana_api_key
+}
+
+data "aws_availability_zones" "available" {}
+
+locals {
+  tenant      = var.tenant      # AWS account name or unique id for tenant
+  environment = var.environment # Environment area eg., preprod or prod
+  zone        = var.zone        # Environment within one sub_tenant or business unit
+
+  region = "us-east-1"
+  azs    = slice(data.aws_availability_zones.available.names, 0, 3)
+
+  cluster_version = "1.21"
+  cluster_name    = join("-", [local.tenant, local.environment, local.zone, "eks"])
+
+  vpc_cidr = "10.0.0.0/16"
+  vpc_name = join("-", [local.tenant, local.environment, local.zone, "vpc"])
+
+  terraform_version = "Terraform v1.1.7"
+}
+
+module "aws_vpc" {
```

no required, but if you could 
```suggestion
module "vpc" {
```